tests/integration/clientv3/ordering_util_test.go

```diff
@@ -78,8 +81,16 @@ func TestEndpointSwitchResolvesViolation(t *testing.T) {
 	if err != ordering.ErrNoGreaterRev {
 		t.Fatal("While speaking to partitioned leader, we should get ErrNoGreaterRev error")
 	}
+
+	clus.Members[2].RecoverPartition(t, clus.Members[:2]...)
+	time.Sleep(1 * time.Second) // give enough time for the operation
```

Comment:
NOTE: The `orderingKv.Get` doesn't have backoff try. It might make the test case flaky since the CI VM is unstable and the 1 second might be not enough for member[2] to get latest data.

> Sorry, I don't have good idea to ensure the data is synced from leader. Maybe we can remove `clientv3.WithSerializable()` option to force to use linear read.

Reply:
I think when creating this orderingKv object, we passed the retry function `NewOrderViolationSwitchEndpointClosure`
```
	orderingKv := ordering.NewKV(cli.KV, ordering.NewOrderViolationSwitchEndpointClosure(cli))
```
when the Get fails, we will enter the retry based on
```
		err = kv.orderViolationFunc(op, r, prevRev)
		if err != nil {
			return nil, err
		}
```
Am I understanding this correctly?

Reply:
Sorry for unclear comment. 

The `orderingKv` did retry immediately. 

However, the Github Action CI VM is unstable. When you recover member[2] from network partition and sleep 1s, it doesn't guarantee the member[2] can sync with leader during ordering's retry. 

The Line 87 `_, err = orderingKv.Get(ctx, "foo", clientv3.WithSerializable())` still might return error.

Reply:
Ah that make sense. The idea is with default linearizable it will be forced to talk to the other members to ensure the most recent data is returned when `Get` is called right?

Removed with `WithSerializable` option from the call. Let me know if looks good